imageai/Detection/Custom/__init__.py

```diff
@@ -214,6 +213,8 @@ def trainModel(self):
         if self.__training_mode:
```

Comment:
Does it make sense to have `self.__training_mode` here, given that this is actually the training method? I mean, this will always evaluate to Tue, right? What's more, this should be set to `True` in this method

Reply:
Thanks for your review of this @rola93 . The property `self.__training_mode` is used to control the behaviour of the class when different tasks are executed e.g value is `False` during evaluation.